jwst/pipeline/outlier_detection.cfg

```diff
@@ -13,3 +13,5 @@ snr = '4.0 3.0'
 scale = '0.5 0.4'
 backg = 0.0        
 save_intermediate_results = False
+resample_data = True
+good_bits = 4
```

Comment:
I see, so good_bits is used to tell the routine to ignore certain conditions on input and in this case it's telling it to ignore anything that was flagged as a jump on a previous pass (so that it starts over from scratch).